src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -1993,6 +1996,22 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
         }).join(".");
     }
 
+    protected async getEnumTypeName(table: Table, column: TableColumn) {
```

Comment:
Postgres type name is used only during schema sync/migration, so I've added a new method instead of adding another property to TableColumn.

I can add JSDoc comment before method definition, but tbh. I'm not much of a fan of that. Comments have a tendency of being forgotten when functionality changes and quickly becomes out of date(and in many cases don't bring much more info than function name - just look at the function above this one).